balloc/balloc.h

```diff
@@ -213,6 +214,26 @@ enum {
 	M0_BALLOC_BUDDY_LOOKUP_MAX = 10,
 };
 
+/*
+ * BALLOC_DEF_BLOCKS_PER_GROUP * (1 << BALLOC_DEF_BLOCK_SHIFT) = 128 MB -->
+ * which equals group size in ext4
+ */
+enum {
```

Comment:
Any reason for moving this definition up.

Reply:
Earlier Shashank also pointed this.  I am not sure, why it is showing in diff. actually there is no movement.